mllib/src/test/scala/org/apache/spark/mllib/clustering/LDASuite.scala

```diff
@@ -115,6 +115,15 @@ class LDASuite extends SparkFunSuite with MLlibTestSparkContext {
       assert(topicDistribution.toArray.sum ~== 1.0 absTol 1e-5)
     }
 
+    val top2TopicsPerDoc = model.topTopicsPerDocument(2).collect()
+    top2TopicsPerDoc.zip(topicDistributions).foreach {
```

Comment:
The order of documents in `top2TopicPerDoc` and `topicDistribution` are not guaranteed (Please correct me if I am wrong). Recommend we sort them before comparing them.